udata/core/dataset/croquemort.py

```diff
@@ -53,7 +65,7 @@ def check_url(url, group=None):
         log.error(ERROR_LOG_MSG, exc_info=True)
         return {}, 503
     attempts = 0
-    while response.status_code == 404 or 'status' not in response.json():
+    while not has_checked(response):
```

Comment:
I modify it, but I use a negation on purpose: I like to enforce that something has not happen or is not done